bcdaworker/main.go

```diff
@@ -204,9 +204,24 @@ func writeBBDataToFile(bb client.APIClient, acoID string, acoCMSID string, cclfB
 	totalBeneIDs := float64(len(cclfBeneficiaryIDs))
 	failThreshold := getFailureThreshold()
 	failed := false
+	suppressedList := models.GetSuppressedBlueButtonIDs()
```

Comment:
I don't think this list is going to be populated with what you want, since the return from this function is a list that was populated when the request was made (hence, no BB IDs in `cclf_beneficiaries` table yet).